partner_delivery_zone/models/res_partner.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    delivery_zone_id = fields.Many2one(
+        comodel_name='partner.delivery.zone',
+        string='Delivery Zone',
+        ondelete='restrict',
+        index=True,
+    )
+
+    @api.model
+    def _commercial_fields(self):
```

Comment:
@sergio-teruel Really? This could be too restrictive as it forbids to have a company with several delivery zones.

Reply:
Uuuupss!!! You are right

Reply:
You can still provide a default when creating the contacts.